### ISSUE TITLE
Deprecation - Telephone React component

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#1631]
## Description

Per team discussion, we are sunsetting our previous phased deprecation strategy in favor of directly migrating components ourselves. 

- [ ] Check if the component is still being used in vets-website
- [ ] Migrate any remaining instances of the React component ourselves.
- [ ] Once there are no more uses, remove the React component from the component-library repo:
- [react-components](https://github.com/department-of-veterans-affairs/component-library/tree/main/packages/react-components) package
- [design system dashboard cli](https://github.com/department-of-veterans-affairs/component-library/tree/main/packages/design-system-dashboard-cli) package
- [storybook](https://github.com/department-of-veterans-affairs/) package
- [ ] Remove any ESLint rules from [vets-website](https://github.com/department-of-veterans-affairs/vets-website) and the [eslint-plugin](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/master/packages/eslint-plugin).
- [ ] Remove it from the [vets-website GA script](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/site-wide/component-library-analytics-setup.js#L67-L73).
- [ ] Remove the related web component(s) from the [web component migrations list](https://design.va.gov/about/developers/using-web-components#web-component-migrations)

Migration steps look like this:

1. Submit PR, let the email go out, wait a day. If no action:
2. @ the team(s) directly with a comment on the PR. Wait a day. If no action:
3. Message the team(s) directly in slack, requesting a review.  Wait a day. If no action:
4. Merge pull requests.

## Tasks
- [ ] React Component deprecation has been completed following the above process

## Acceptance Criteria
- [ ] Deprecated component has either been removed entirely (if no longer in use) or moved to the next stage of the deprecation process.

**Comment by micahchiang** [comment]
Here are some things on deprecating the Telephone Component.

**TLDR**: A lot of the import instances can be boiled down to 1 of 2 things:
1. Importing `CONTACTS` from the old Telephone component, which can now just be imported directly as mentioned below.
2. Repeat usage of `GetFormHelp` copy and pasted into multiple applications. 

`va-telephone` is already used pretty extensively in vets-website, so I feel pretty confident that we could probably get away with a large find and replace without introducing visual regression.

- Generally, if a migration from `<Telephone />` to `<va-telephone>` can be verified to introduce no visual change or regression in an unauthenticated use case, I wouldn't necessarily worry about testing every single usage that requires you to be logged in locally
- Many of the instances where `<Telephone />` is being used are in components that are copied and pasted from one application to another (`GetFormHelp` is maybe the most common offender). These don't necessarily need to be tested individually.
- If a component imports `CONTACTS` from the Telephone component, those can be imported directly instead with: `import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';`
-  `/platform/user`: Import contacts like above and just replace Telephone with `va-telephone`
- `/platform/static-data`: 
  - [SubmitSignInForm](https://github.com/department-of-veterans-affairs/vets-website/blob/ce7c624b3de58cdcc77b32324a6bb892a59d8f9c/src/platform/static-data/SubmitSignInForm.jsx#L4): Import contacts directly. Can be tested by going to `localhost:3001/terms-of-use` and verifying phone number looks the same. [terms of use instance](https://github.com/department-of-veterans-affairs/vets-website/blob/ce7c624b3de58cdcc77b32324a6bb892a59d8f9c/src/applications/terms-of-use/containers/TermsOfUse.jsx#L80) or by clicking on the login button locally
  - [LoginBanners](https://github.com/department-of-veterans-affairs/vets-website/blob/58c33cc05b428743579f1918b6035cebf9b6eecc/src/platform/user/authentication/components/LoginBanners.jsx#L42): Import contacts directly (This component isn't imported and used anywhere :shrug:)
  -  `/applications/vre`:
	  - If you see a questionnaire, click the link that says `Apply online` to skip it
	  - May be able to skip filling out the form by visiting http://localhost:3001/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/confirmation
	  - Fill out the forms to verify updates look the same. Ensure vets-api is running. These can be filled out without logging in locally
		  - Check the confirmation
  - `/applications/static-pages/health-care-manage-benefits`:
	  - Import contacts directly
  - `/static-pages/cta-widget/components/messages/NeedsVAPatient`: Not imported anywhere :shrug:
  - `/applications/pre-need`: visit `localhost:3001/burials-and-memorials/pre-need/form-10007-apply-for-eligibility` and scroll to the Get Help section at the bottom of the page to verify changes
  - `/applications/personalization`:
	  - Login locally using the directions for [mocked authentication](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity/Mocked%20Authentication)
	  - In applications with Telephone usage, use the `rootUrl` in `manifest.json` to visit the page locally and verify the change
  - `/applications/medical-copays`: import contacts directly
  - `/applications/gi/components/vet-tec/VetTecApplicationProcess`: Not imported anywhere other than corresponding unit test
  - `/applications/financial-status-report`: uses react bindings for va-telephone, contacts need to be imported directly
  - `/applications/facility-locator`: import contacts directly
  - `/applications/edu-benefits`:
	  - `GetFormHelp` should be available at the `rootUrl` of any of the edu forms
